config/axios.js

```diff
@@ -0,0 +1,9 @@
+const axios = require('axios');
+
+const api = axios.create({
+  baseURL: process.env.EXTERNAL_API,
```

Comment:
Env variables names should be more descriptive. In a large project you may have a lot of envs and several external APIs, so maybe something like this `{external_api_name}_URL`